### PR TITLE
change r2dbc to jdbc

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: src/test/resources/application-default.yml [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+logging:
+  level:
+    org.springframework.orm.jpa: DEBUG
+    org.springframework.transaction: DEBUG
+    org.hibernate.SQL: DEBUG
+    org.hibernate.type.descriptor.sql.BasicBinder: TRACE
+
 dbVendor: h2:mem
 query: ;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
 hostPort: ":///"
```
Comment (FranckLecuyer) [inline src/test/resources/application-default.yml:19]: hostPort: ":"   like in another postgres services (study-server, network-modification-server ...) ?

File: src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java [hunk 2]
```diff
@@ -88,6 +82,8 @@
     @MockBean
     private NetworkStoreService networkStoreClient;
 
+    //@Autowired
+    //private DynamicSimulationWorkerService dynamicSimulationWorkerServiceAutowired;
```
Comment (FranckLecuyer) [inline src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java:86]: Suppress these commented lines ?

File: src/test/resources/application-default.yml (hunk 1 above)
Comment (JacquesBorsenberger) [reply src/test/resources/application-default.yml]: done

File: src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java [hunk 3]
```diff
@@ -100,15 +94,11 @@
     @Before
     public void init() throws IOException {
         MockitoAnnotations.initMocks(this);
-
+       // dynamicSimulationWorkerService = Mockito.spy(dynamicSimulationWorkerServiceAutowired);
```
Comment (jonenst) [inline src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java:97]: why comment this ?

File: src/main/java/org/gridsuite/ds/server/service/DynamicSimulationWorkerService.java [hunk 4]
```diff
@@ -68,6 +69,10 @@
     @Autowired
     private StreamBridge publishResult;
 
+    @Lazy // lazy because test use spybean on DynamicSimulationWorkerService
```
Comment (jonenst) [inline src/main/java/org/gridsuite/ds/server/service/DynamicSimulationWorkerService.java:72]: is this really needed ?

File: src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java (hunk 3 above)
Comment (JacquesBorsenberger) [reply src/test/java/org/gridsuite/ds/server/DynamicSimulationTest.java]: removed

**Review by jonenst — APPROVED** [review]

**Pull request merged**